### PR TITLE
Updating our UI to be more focused and centered so that player can see it/understand it easier

### PR DESCRIPTION
**Pull request by scottnm**
the new ui

### CODE REVIEW

File: Assets/Scripts/CoinsUIController.cs [hunk 1]
```diff
@@ -46,7 +46,10 @@ void Start()
...
             for (int i = 0; i < GameConstants.MAX_ONLINE_PLAYERS_IN_GAME; ++i)
             {
-                mVoteUIElements[i].SetActive(activePlayers[i]);
+                if (!activePlayers[i])
+                {
+                    DisableUI(i);
+                }
             }
 
             StartCoroutine(HideInstructions());
```
Comment (phoenixbishea) [inline Assets/Scripts/CoinsUIController.cs]: Should we remove Instructions or at least disable them for the time being?

File: Assets/Scripts/CoinsUIController.cs [hunk 2]
```diff
@@ -79,7 +82,39 @@ public void UpdateCoinCounts(int ownerId, int newCoinBalance)
...
+        {
+            foreach (Image img in mVoteUIElements[playerNum].GetComponentsInChildren<Image>())
+            {
+                Color c = img.color;
+                c.a = 0.2F;
+                img.color = c;
+            }
+            foreach (Text text in mVoteUIElements[playerNum].GetComponentsInChildren<Text>())
+            {
+                Color c = text.color;
```
Comment (phoenixbishea) [inline Assets/Scripts/CoinsUIController.cs]: Do you have default values of 0 for all of the text elements?

File: Assets/Scripts/CoinsUIController.cs (hunk 1 above)
Comment (scottnm) [inline Assets/Scripts/CoinsUIController.cs]: For now it's harmless and will probably be ignored so I wouldn't hold up this PR for that to get taken care of. If you think it hurts the final build we can strip it out in another PR

**Comment by phoenixbishea** [comment]
Did you remove all of the space between the UI elements?

File: Assets/Scripts/CoinsUIController.cs (hunk 2 above)
Comment (scottnm) [inline Assets/Scripts/CoinsUIController.cs]: Yes The text defaults at 0

**Pull request merged**